### PR TITLE
Use studio "host" in state, and accept "host" from api

### PR DESCRIPTION
**Pull request by seotts**
This switches over to using "host" (instead of "owner") terminology in the redux state for studio. It also allows www to accept a field called "host" from api. It falls back on "owner," so this can be merged anytime before the change is made in [api](https://github.com/LLK/scratch-api/pull/1669).

There should be no changes to functionality as a result of this change.

### CODE REVIEW

File: src/redux/studio-permissions.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ const selectCanTransfer = (state, managerId) => {
     if (state.studio.classroomId !== null) return false;
     if (selectIsMuted(state)) return false; // Muted users cannot transfer studios.
     if (state.studio.managers > 1) { // If there is more than one manager,
-        if (managerId === state.studio.owner) { // and the selected manager is the current owner/host,
+        if (managerId === state.studio.host) { // and the selected manager is the current owner/host,
```
Comment (ericrosenbaum) [inline src/redux/studio-permissions.js:64]: this comment should be updated I think?

File: src/redux/studio-permissions.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ const selectCanRemoveProject = (state, creatorUsername, actorId) => {
 
     // Admins/managers can remove any projects
     if (isManager(state) || selectIsAdmin(state)) return true;
-    // Project owners can always remove their projects
+    // Project hosts can always remove their projects
```
Comment (ericrosenbaum) [inline src/redux/studio-permissions.js:77]: this change seems incorrect

Comment (seotts) [reply src/redux/studio-permissions.js:77]: Oh good catch!

**Comment by apple502j** [comment]
Will the API still return `owner` key, or will it be removed?

**Review by ericrosenbaum — APPROVED** [review]
LGTM!

**Comment by Naleksuh** [comment]
@seotts You replaced all instances of "owner" with "host', but what about references to a studio "creator"? Will these become host as well?

**Pull request merged**